projects/aca-content/folder-rules/src/rule-details/rule-details.ui-component.scss

```diff
@@ -33,6 +33,10 @@
       align-items: baseline;
       gap: 8px;
 
+      #{$mat-floating-label}#{$mat-floating-label-above} {
```

Comment:
this does not make sense, this is still indirect dependency on the material selectors imho

Reply:
Ive updated the selector to this - 
```
label[for="rule-details-name-input"] {
  transform: translateY(-20%) scale(1);
}
```
However, the strategy of using selectors via the mat-selectors.scss file is something that was done throughout the angular upgrade process. If we want to replace most/all of those occurences, then we should probably open create a ticket for that task